addons/backgrounds/package.json

```diff
@@ -27,7 +27,6 @@
     "@storybook/addons": "4.0.0-alpha.10",
     "@storybook/core-events": "4.0.0-alpha.10",
     "babel-runtime": "^6.26.0",
-    "emotion": "^9.1.3",
```

Comment:
When you remove emotion, what's left is abstract emptiness. Mah hart. Mah sole...

Reply:
💔